developer/js/source/lexical-model-compiler/lexical-model.ts

```diff
@@ -23,13 +23,47 @@ interface WordBreakerSpec {
    *    joinWordsAt: ['-'] // to keep hyphenated items together
    */
   readonly joinWordsAt?: string[];
+
+  /**
+   * Overrides word splitting behaviour for certain scripts.
+   * For example, specifing that spaces break words in scripts that do not
+   * traditionally use spaces.
+   */
```

Comment:
"Currently only supported for scripts based in SE Asia, but may be extended for others as needed."

Reply:
I put this comment lower down in the file: https://github.com/keymanapp/keyman/pull/3506/files#diff-ce42e5e080cc5925ee8811a51b1fcb77R62